doc/docusaurus/docs/using-plutus-tx/running-debugging.md

```diff
@@ -0,0 +1,301 @@
+  -h             --help               Show usage
+  -V             --version            Show version
+  -q             --quiet              Don't print text (error) output; rely only on exit codes
+  -v             --verbose            Print more than than the default
+                 --print-builtins     Print the Default universe & builtins
+                 --print-cost-model   Print the cost model of latest Plutus Version as JSON
+                 --stdin              Use stdin
+  -e[NAME]       --example[=NAME]     Use example NAME as input. Leave out NAME to see the list of examples' names
+  -p STYLE       --pretty=STYLE       Make program's textual-output&error output pretty. Ignored for non-textual output (flat/cbor). Values: `classic`, `readable, `classic-simple`, `readable-simple`
+  -o FILE                             Write compiled program to file
+                 --stdout             Write compiled program to stdout
+  -O[INT]                             Set optimisation level; default: 0 , safe optimisations: 1, >=2: unsafe optimisations
+                 --whole-opt          Run an extra optimisation pass after all inputs are applied together. Ignored if only 1 input given.
+  -x SUFFIX                           Causes all files following this option on the command line to be processed as if they had the suffix SUFFIX
+  -n NAMING      --nam=NAMING         Change naming to `name` (default), `debruijn` or `named-debruijn`
+  -a ANNOTATION  --ann=ANNOTATION     Change annotation to `unit` (default) or `srcspan`
+                 --run                Compile and run
+                 --bench[=SECS]       Compile then run repeatedly up to these number of seconds (default:10) and print statistics
+                 --debug[=INTERFACE]  Compile then Debug program after compilation. Uses a `tui` (default) or a `cli` interface.
+                 --debug-dir[=DIR]    When `--debug`, try to search for PlutusTx source files in given DIR (default: .)
+                 --budget=INT,INT     Set CPU,MEM budget limit. The default is no limit. Only if --run, --bench, or --debug is given
+```
+
+## Compiling with the `plutus` CLI
+
+Before you can even run or debug your code, you first need to compile and extract the plutus code from the high-level source code.
+The steps for extracting plutus code vary depending on the source language you are using (Plutus Tx, Aiken, ...);
+if your are using Plutus Tx as source, you can follow the instructions on [how to extract compiled code].
+
+Although the CLI tool cannot help with compiling high-level code (e.g. Plutus Tx), it can aid with compiling and converting
+between different formats and intermediate representations of plutus code.
+
+In the following example we check a program written in the PIR (Plutus Intermediate Representation) language:
+
+``` shell
+$ echo "(program 1.1.0 (lam x (con integer) (lam y (con integer) x)))" > const.pir
+$ plutus const.pir
```

Comment:
So if you run `plutus` without any flag, it performs static checks? That's a weird default, and isn't clear from the help message.